src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -3487,7 +3488,8 @@ void wm_vuldet_adapt_title(char *title, char *cve) {
         offset += strlen(cve) + 1;
     }
     os_strdup(title + offset, title_ofs);
-    strncpy(title, title_ofs, strlen(title_ofs) + 1);
+    strncpy(title, title_ofs, size_title);
+    title[size_title] = '\0';
```

Comment:
This is a false positive due title_ofs is a part of title, meaning is shorter than title and using strlen as n of strncpy is OK. But in terms of correctness and the fact that n means to protect the dest, this is OK, but i suggest to change it to the next approach
```suggestion
    snprintf(title, title_size, "%s", title_ofs);
```